pyproject.toml

```diff
@@ -228,15 +225,15 @@ showcontent = true
 
 [tool.coverage.run]
 branch = true
-source = ["trio"]
+source_pkgs = ["trio"]
 omit = [
     # These are run in subprocesses, but still don't work. We follow
     # coverage's documentation to no avail.
-    "*/trio/_core/_tests/test_multierror_scripts/*",
+    "*/src/trio/_core/_tests/test_multierror_scripts/*",
     # Omit the generated files in trio/_core starting with _generated_
-    "*/trio/_core/_generated_*",
+    "*/src/trio/_core/_generated_*",
     # Script used to check type completeness that isn't run in tests
-    "*/trio/_tests/check_type_completeness.py",
+    "*/src/trio/_tests/check_type_completeness.py",
```

Comment:
I presume these /src/ changes are for source code? But don't we run coverage against trio as a package? I hope we're not shipping /src/ in the wheel...

Reply:
Should be fixed in 3b582ca